app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="docs-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
When the user on cloud hosting, it should open the cloud hosting page content /public/documentation/cloud_hosting/index.md

![image](https://user-images.githubusercontent.com/8007722/71929029-b7e6c180-3166-11ea-8d9f-e8542427c712.png)

Reply:
Done, but the accordion functionality is gone.